src/mesh/mod.rs

```diff
@@ -30,24 +28,18 @@ pub fn build_gizmo(
     let plane_size = axis_length * 0.25;
     let plane_offset = plane_size / 2. + axis_length * 0.2;
     // Define gizmo meshes
-    let arrow_tail_mesh = meshes.add(Mesh::from(shape::Capsule {
+    let arrow_tail_mesh = meshes.add(Mesh::from(Capsule3d {
         radius: 0.04,
-        depth: axis_length,
+        half_length: axis_length * 0.5f32,
         ..Default::default()
     }));
     let cone_mesh = meshes.add(Mesh::from(cone::Cone {
         height: 0.25,
         radius: 0.10,
         ..Default::default()
     }));
-    let plane_mesh = meshes.add(Mesh::from(shape::Plane::from_size(plane_size)));
-    let sphere_mesh = meshes.add(
-        Mesh::try_from(shape::Icosphere {
-            radius: 0.2,
-            subdivisions: 3,
-        })
-        .unwrap(),
-    );
+    let plane_mesh = meshes.add(Mesh::from(Plane3d::default()));
```

Comment:
The new default plane is too big for the gizmo.  The below modified line should fix this:

`let plane_mesh = meshes.add(Mesh::from(Plane3d::default().mesh().size(plane_size, plane_size)));`

Reply:
Updated in ccf615d